dipy/denoise/noise_estimate.py

```diff
@@ -18,13 +18,14 @@ def _inv_nchi_cdf(N, K, alpha):
                 2: 0.7306303027491917,
                 4: 0.6721952960782169,
                 8: 0.6254030432343569,
-               16: 0.5900487123737876,
-               32: 0.5641772300866416,
-               64: 0.5455611840489607,
-              128: 0.5322811923303339}
+                16: 0.5900487123737876,
```

Comment:
Those were intentionally not aligned, so that the decimal point aligns instead of the blank space.